### DISCUSSION TITLE
hasak and linHPSDR working together

### DISCUSSION THREAD

**Discussion by m5evt** [softerhardware/CWKeyer#15] [Show and tell]
My linHPSDR fork (with a [MIDI branch for now](https://github.com/m5evt/linhpsdr/tree/midi)) is working nicely with the hasak firmware.

I'm going to make an effort to have a few CW qsos each day with it for the next week to stress test the combination.

![linhpsdr](https://user-images.githubusercontent.com/60316466/153709536-b2c4c17f-dba6-4384-8003-c60c5e60432a.png)

**Comment by dl1ycf** [comment]
Perhaps you should rename CWRIGHT to CWKEYER, since CWRIGHT/CWLEFT could be reserved if you want to build in a iambic keyer in the future. However, projects as the TeensyKeyer have made built-in iambic keyers mostly obsolete.
